howto/monitor/landing.md

```diff
@@ -1,3 +1,12 @@
-The guides in this section describe how to monitor your Anbox Cloud or Anbox Cloud Appliance installation.
+Anbox Cloud gathers various performance metrics and makes them accessible through API endpoints. While Anbox Cloud does not provide its own observability solution, it supports implementing and integrating custom solutions for monitoring performance.
 
-To check which metrics are available for monitoring, see [Prometheus metrics](https://discourse.ubuntu.com/t/prometheus-metrics/19521).
+See [Prometheus metrics](https://discourse.ubuntu.com/t/prometheus-metrics/19521) for a list of available metrics and how to access them. [LXD Metric exporter for instances](https://discuss.linuxcontainers.org/t/lxd-metric-exporter-for-instances/11735) has a list of metrics available through LXD.
```

Comment:
These links contain the metrics that are available. But is this sufficient for users that implement their own monitoring solution to figure out how to retrieve them?

Reply:
I think we need to add some details to the reference page on how the metrics from each service can be retrieved via `curl` for example.

Reply:
We have AC-701 to provide that